Changes.md

```diff
@@ -1,7 +1,10 @@
 1.6.x.x (relative to 1.6.2.1)
 =======
 
+Features
+--------
 
+- TaskNode : Added `isolated` plug. When an isolated task is dispatched, a script containing only the task node will be saved for each dispatched batch and all plug inputs will be computed at dispatch time. For tasks don't require varying the context to compute any of their inputs, this can greatly decrease load times for large scripts. It can also reduce execution time by shifting plug computation to the dispatching process, which can make use of cached plug values when possible.
```

Comment:
If we go with the metadata idea, let's replace "TaskNode" with "SystemCommand, PythonCommand". And also add an API entry explaining the metadata so folks can opt their own nodes in as desired.

I would again replace the "For tasks don't require varying the context" stuff with a simpler explanation of the benefits - maybe the one from my description metadata suggestion.

We also still need to add the reference to the original issue.

Reply:
I changed the wording in https://github.com/GafferHQ/gaffer/pull/6612/commits/898c46c2f7324c13dc1210c22ccf4d22c884bb71, https://github.com/GafferHQ/gaffer/pull/6612/commits/9ce45d4b33dbf1f7803feedc2d0b683522540496 and 6328821a397de4b0e9e0c438298e2b5b662be04b.